index-handlers/src/test/java/no/sikt/nva/nvi/index/apigateway/FetchInstitutionReportHandlerTest.java

```diff
@@ -150,6 +155,33 @@ void shouldReturnCandidatesWithApprovalsBelongingToUsersTopLevelOrganization(Str
         assertEquals(expected, actual);
     }
 
+    @ParameterizedTest
+    @MethodSource("listSupportedLanguages")
+    void shouldReturnReportWithLanguageLabel(String languageUri, String expectedLabel) throws IOException {
```

Comment:
Not keen on providing both input and expected in this way, but I see the reason.

Reply:
Why not?

Reply:
Thank you for explaination and examples @brinxmat. I removed expected as input in latest commit